_data/openseeds/ols-9/schedule.yaml

```diff
@@ -0,0 +1,555 @@
+        your project: [https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/)\n
+        * Link your README in a comment on the OLS-8 issue\n* Add an open license
+        to your repository as a file called `LICENSE.md: [https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/)\n
+        * Add a Code of Conduct to your repository as a file called `CODE_OF_CONDUCT.md:
+        [https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/)"
+      talks:
+      - slides: 
+          https://docs.google.com/presentation/d/e/2PACX-1vT6Xn-rv8tA_YttEhJoO_aqOx36REJVZ9l44Orz69sm8y1bxuiC-HLj41-e7QAAw-drhPmucehO-Clv/pub
+        title: Setting up a project
+        tag: Setting up a project
+      - slides: 
+          https://docs.google.com/presentation/d/16xlUE0NW7W4hW2PBMs89bNcXM9pqcmM9WnuAEgqo3DM/edit?usp=sharing
+        title: 'ReadMe files: how to make your project welcoming to others'
+        tag: README
+      - slides: 
+          https://docs.google.com/presentation/d/e/2PACX-1vQsH4kNQ838uk6mHw8w6YuR8hgg4JXZ10lkNdaVC05zfMEZpU1xDRyT_B6A3s9rqirP2RhAndi-G6dd/pub
+        title: A Primer on Open Licenses
+        tag: Open Licensing
+      - slides: 
+          https://docs.google.com/presentation/d/1E0HBEzzuO3VjNLwM3ThWLJ27R6h359j0DO4jwfcFkCw/edit#slide=id.g21280744133_2_56
+        title: Designing and enforcing a Code of Conduct
+        tag: Code of Conduct
+  '05':
+    start: August 08, 2024
+    calls:
+    - duration: 30 min
+      title: 3rd mentor-mentee meeting
+      type: Mentor-Mentee
+      talks: []
+    - date: August 09, 2024
+      time: "09:00"
+      duration: 90 min
+      notes: https://bit.ly/ols-9-week-05
+      title: GitHub tutorial for beginners
+      recording: https://youtu.be/Af-VLs_9DXo
+      type: Skill-up
+      learning_objectives:
+      - "Create a new repository\n* Create a (README) file\n* Write in Markdown\n
+        * Manage issues and labels\n* Enable a small website via GitHub\n* Submit
+        changes to a GitHub repository via Pull Request (PR)\n* Set the scene for
+        self-paced learning after this session"
+      before:
+      - "Please create a GitHub account: [https://github.com](https://github.com)\n
+        * Have a look at the OLS-8 repository for 'project issues': [https://github.com/open-life-science/ols-8/issues](https://github.com/open-life-science/ols-8/issues)\n
```

Comment:
The spreadsheet needs to be updated to remove mention of OLS-8 in there and then you can re-run the script so that it gets updated here